src/Exception/InvalidParameterException.php

```diff
@@ -13,11 +13,9 @@
 
 namespace Sonata\NotificationBundle\Exception;
 
-use Sonata\CoreBundle\Exception\InvalidParameterException as CoreBundleException;
-
 /**
- * @deprecated Use Sonata\CoreBundle\Exception\InvalidParameterException instead
+ * @deprecated Use \RuntimeException instead
```

Comment:
Not sure about this one, wdyt @sonata-project/contributors ?

Reply:
We should maybe remove the deprecation